src/content/contributing/style-guide/index.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Style Guide
+description: Style Guide for ethereum.org
+lang: en
+sidebar: true
+---
+
+# Ethereum.org Style Guide {#style-guide}
+
+Content on [Etheruem.org](http://etheruem.org) is crowdsourced and primarily written by our incredible contributors. This style guide aims to standardize certain aspects of contributing content to make the contribution process smoother.
+
+## Who can submit content to Ethereum.org {#who-can-help}
+
+Anyone! [Ethereum.org](http://ethereum.org) is entirely open source, and many of its best pages are submitted by curious learners who expanded their notes into documentation pages now living on the site!
+
+## Audience {#audience}
+
+[Ethereum.org](http://ethereum.org) content caters to a large and varied audience. Loosely we can categorize them as:
+
+#### Individuals
+
+App users, investors, enthusiasts, people who are "new to Ethereum"
+
+#### Developers
+
+Developers or others who want technical information about Ethereum.
+
+#### Enterprises
+
+People, businesses, and other organizations who want to understand Ethereum's value in an enterprise setting
+
+Because [Ethereum.org](http://ethereum.org) caters to a wide variety of users, all content on the site should make an effort to explain any technologies and concepts as simply as possible.
```

Comment:
Would keep ethereum.org lowercased if not starting a sentence:
```suggestion
Because ethereum.org caters to a wide variety of users, all content on the site should make an effort to explain any technologies and concepts as simply as possible.
```
or 
```suggestion
Because [ethereum.org](/) caters to a wide variety of users, all content on the site should make an effort to explain any technologies and concepts as simply as possible.
```